scripts/travis-extract-l10n.sh

```diff
@@ -33,7 +33,7 @@ function init_environment {
     git checkout master
     git checkout -b "$BRANCH_NAME"
 
-    make -f Makefile-docker install_python_dependencies
+    make -f Makefile-docker install_python_test_dependencies
```

Comment:
This may work using `install_python_common_dependencies` but really, this is run once a week so no need to optimize I think.

Reply:
`install_python_common_dependencies` is just `system.txt`.  We could install the `prod.txt` dependencies directly, or add a makefile rule to do that rather than `tests.txt`, but it's not really worth it as you mention.